frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -44,14 +69,42 @@ export function compareActivities(a, b) {
 /**
  * Get the field of field name `fieldName` from `activity` or the default value.
  * 
- * @param {Object} activity 
- * @param {string} fieldName 
- * @param defaultValue 
- * @returns `activity[fieldName]` if possible, else `defaultValue`.
+ * @param {ActivityInfo} activity The activity from which to get the field.
+ * @param {string} fieldName Name of field to get.
+ * @param {*} defaultValue Value if field is not found/is null.
+ * @returns {*} <code>activity[fieldName]</code> if possible, else <code>defaultValue</code>.
  */
 export function getField(activity, fieldName, defaultValue) {
   if (activity[fieldName] === null || activity[fieldName] === undefined) {
     return defaultValue;
   }
   return activity[fieldName];
+}
+
+/**
+ * Write contents into an activity already existing in the database.
+ * 
+ * @param {string} tripId Database ID of the trip whose actiivty should be modified.
+ * @param {string} activityId Database ID of the activity to be modified.
+ * @param {Object} newValues Dictionary of the new values in <code>{fieldName: newValue}</code> form
+ * @returns {boolean} <code>true</code> if the write was successful, <code>false</code> otherwise. 
```

Comment:
Is this return value being used anywhere?

Reply:
not officially, no. could be used for error messages later though